app/services/booking_bug_connection.rb

```diff
@@ -7,23 +7,21 @@ def initialize(config:)
   end
 
   def auth_token
+    Rails.logger.info('BookingBugConnection') { 'Retrieving authorisation token' }
```

Comment:
We don't have to run the application with a debug log level - I would consider setting it to info if the logs are too verbose.

Given these processes are going to only be called daily I don't think we're going to have a problem of too many log entries.